.sourcery.yaml

```
@@ -0,0 +1,144 @@
+rule_settings:
+  enable:
+    - aug-assign
+    - bin-op-identity
+    # - break-or-continue-outside-loop (without autofix?)
+    - chain-compares
+    - collection-into-set
+    - collection-to-bool
+    - compare-via-equals
+    - convert-any-to-in
+    - convert-to-enumerate
+    - dataframe-append-to-concat
+    - de-morgan
+    - default-get
+    # - default-mutable-arg (todo: disable ruff B006)
+    - del-comprehension
+    - dict-assign-update-to-union
+    - dict-comprehension
+    # - ensure-file-closed (todo: disable ruff sim115)
+    - flatten-nested-try
+    - for-append-to-extend
+    - for-index-replacement
+    - for-index-underscore
+    - guard
+    - hoist-loop-from-if
+    - hoist-repeated-if-condition
+    - hoist-similar-statement-from-if
+    - hoist-statement-from-if
+    - hoist-statement-from-loop
+    - instance-method-first-arg-name
+    - invert-any-all
+    - invert-any-all-body
+    - last-if-guard
+    - lift-duplicated-conditional
+    - lift-return-into-if
+    - max-min-default
+    - merge-assign-and-aug-assign
+    - merge-comparisons
+    - merge-dict-assign
+    - merge-duplicate-blocks
+    # - merge-else-if-into-elif (todo: disable ruff prl501)
+    - merge-except-handler
+    - merge-list-append
+    - merge-list-appends-into-extend
+    - merge-list-extend
+    - merge-nested-ifs
+    - merge-repeated-ifs
+    - merge-set-add
+    - min-max-identity
+    - missing-dict-items
+    - move-assign
+    - move-assign-in-block
+    - non-equal-comparison
+    - path-read
+    - raise-from-previous-error
```

```suggestion
    # - raise-from-previous-error
```
@dorschw i found it to be very annoying in content, especially when raising DemistoException, which is when we override the exception to tell the user something more meaningful.

---

`raise from` both the stack trace of the original exception, and the custom message